docs/index.md

```diff
@@ -35,7 +35,15 @@ For smaller sites (less than 50 worker nodes), the most common way to add a site
 Add OSG Software
 ----------------
 
-If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.  The general steps to installing an OSG site are:
+
+1. [Retrive a Host Certificate](#installing-and-managing-certificates-for-site-security) for your CE.
+1. Install [OSG Yum Repos](release/yum-basics) and the [Compute Element software](#installing-and-configuring-the-compute-element) on your CE host
+1. Install [OSG Yum Repos](release/yum-basics) and the [Worker Node client](#adding-osg-software-to-worker-nodes) on your worker nodes.
```

Comment:
They don't necessarily need the yum repos on the WNs if they're doing tarball installs